experiments/calibration/observationseries_era5.jl

```diff
@@ -0,0 +1,119 @@
+using ClimaAnalysis
+using ClimaLand
+using ClimaLand.Artifacts
+using ClimaCore
+using Statistics
+import EnsembleKalmanProcesses as EKP
+using LinearAlgebra
+FT = Float64
+
+# Load ERA5 data
+era5_path = joinpath(
+    ClimaLand.Artifacts.era5_surface_data_1979_2024_path(),
+    "era5_monthly_averages_surface_single_level_197901-202410.nc",
+)
+
+# Get era5 data
+vars_temp = Dict(
+    var => OutputVar(era5_path, varname) for (var, varname) in zip(
+        [:lhf, :shf, :swu, :lwu],
+        ["mslhf", "msshf", "msuwswrf", "msuwlwrf"],
+    )
+)
+
+# Get ClimaLand data (for regridding)
+simdir = SimDir(
+    "calibration_output_utki_sample/iteration_000/member_001/global_diagnostics/output_active/",
+)
+out_vars = Dict(var => get(simdir, var) for var in ["lhf", "swu", "lwu", "shf"])
+
+# Resample variables to diagnostic grid
+# note: this is very slow. is it normal?
+vars = Dict()
+for var in keys(vars_temp)
+    vars[var] = ClimaAnalysis.resampled_as(
+        vars_temp[var],
+        out_vars[string(var)],
+        dim_names = "longitude",
+    )
+end
+
+# Extract seasonal data
+results = Dict(
+    :all_seasons => Dict(:lhf => [], :shf => [], :swu => [], :lwu => []),
+    :seasonal_vars => Dict(:lhf => [], :shf => [], :swu => [], :lwu => []),
+    :seasonal_means => Dict(:lhf => [], :shf => [], :swu => [], :lwu => []),
+)
+
+for (lon, lat) in training_locations
+    slices = Dict(
+        var => slice(vars[var], longitude = lon, latitude = lat).data for
+        var in [:lhf, :shf, :swu, :lwu]
+    )
+    slices[:lhf] .*= -1
+    slices[:shf] .*= -1
+    seasonal = Dict(
```

Comment:
maybe this could also be replaced by a ClimaAnalysis function for season mean (if it exists yet)